### ISSUE TITLE
TypeError: this.state._Input.setState is not a function

### ISSUE THREAD

**Issue by riverliway** [antoniopresto/antd-mask-input#41]
* **I'm submitting a ...**
[x] bug report
[ ] feature request
[ ] question about the decisions made in the repository
[ ] question about how to use this project

* **Summary**
When loading the `<MaskedInput>` component, the page instantly crashes and provides me the following information:
```
TypeError: this.state._Input.setState is not a function
    at MaskedInput.setInputValue (MaskedInput.tsx:288:23)
    at MaskedInput.componentDidMount (MaskedInput.tsx:55:10)
    at commitLifeCycles (react-dom.development.js:20663:24)
    at commitLayoutEffects (react-dom.development.js:23426:7)
    at HTMLUnknownElement.callCallback2 (react-dom.development.js:3945:14)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:3994:16)
    at invokeGuardedCallback (react-dom.development.js:4056:31)
    at commitRootImpl (react-dom.development.js:23151:9)
    at unstable_runWithPriority (scheduler.development.js:468:12)
    at runWithPriority$1 (react-dom.development.js:11276:10)
```

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)
I have been doing a lot of overdue infrastructure upgrades, I'm not certain which one broke this component. Some relevant ones might be:
* `antd-mask-input@0.1.13` -> `antd-mask-input@0.1.15`
* Upgraded to `antd@4.19.2`
* Upgraded to `react@17.0.2`

**Comment by rfbasso** [comment]
Same bug here.. This bug is making me crazy, I'm debuging this for hours, and can't find the problem. Tried reinstall node_modules, clone project in a new directory, reinstall node and npm.. none of this works.

**Comment by antoniopresto** [comment]
I'm working on a solution, 90% ready

**Comment by singleDeveloper63** [comment]
> I'm working on a solution, 90% ready

When it will be ready ?

**Comment by victorcesar00** [comment]
Having the same thing here.. What's weird is that this doesn't happend in development, just in production.

**Comment by rfbasso** [comment]
I had found a provisory solution.. The bug don't crashes the page with ANTD 4.18.9
1 - remove package-lock.json
2 - change antd version to 4.18.9 package.json antd version to 4.18.9 (without ^, to get the exact version)
`...,
"antd": "4.18.9",
...`
3 - re-install the packages with npm install --save or yarn install

**Comment by amanullahtanweer** [comment]
Waiting for a fix please

**Comment by singleDeveloper63** [comment]
Unfortunately , this is the only formatting package for antd inputs. We need to contribute to improve

**Comment by tlt254** [comment]
The same problem

**Comment by yangricardo** [comment]
had the same issue now

**Comment by antoniopresto** [comment]
I just published version 2.0
See examples in tests, readme or storybook: https://github.com/antoniopresto/antd-mask-input/blob/master/stories/input.stories.tsx
